### ISSUE TITLE
hadoop: yarn-env.sh template

### ISSUE THREAD

**Issue by wdavidw** [ryba-io/ryba#36]

**Comment by IDerr** [comment]
Still in news ? I think we should review all issues before 2016...

**Comment by wdavidw** [comment]
Yes we can. Just before that, i wish Pierre would remove all template duplicates he created when introducing .j2

**Comment by wdavidw** [comment]
For this, a new isuue could be open after closing that one.
